Sources/EpoxyCollectionView/CollectionView/ReusableViews/CollectionViewCell.swift

```diff
@@ -76,27 +74,29 @@ public final class CollectionViewCell: UICollectionViewCell, ItemCellView {
       let horizontalFittingPriority = fittingPrioritiesProvider.horizontalFittingPriority
       let verticalFittingPriority = fittingPrioritiesProvider.verticalFittingPriority
 
-      // In some cases, `contentView`'s required width and height constraints
-      // (created from its auto-resizing mask) will not have the correct constants before invoking
-      // `systemLayoutSizeFitting(...)`, causing the cell to size incorrectly. This seems to be a
-      // UIKit bug.
-      // https://openradar.appspot.com/radar?id=5025850143539200
-      // The issue seems most common when the collection view's bounds change (on rotation).
-      // We correct for this by updating `contentView.bounds`, which updates the constants used by
-      // the width and height constraints created by the `contentView`'s auto-resizing mask.
-
-      if
-        horizontalFittingPriority == .required &&
-        contentView.bounds.width != layoutAttributes.size.width
-      {
-        contentView.bounds.size.width = layoutAttributes.size.width
-      }
-
-      if
-        verticalFittingPriority == .required &&
-        contentView.bounds.height != layoutAttributes.size.height
-      {
-        contentView.bounds.size.height = layoutAttributes.size.height
+      if #unavailable(iOS 14.0) {
```

Comment:
Awesome this was fixed by Apple!

I think we need to use `#available` since this is a new condition only on Swift 5.6 https://www.hackingwithswift.com/swift/5.6/unavailable